pkg/apis/acid.zalan.do/v1/postgresql_type.go

```diff
@@ -127,3 +128,9 @@ type UserFlags []string
 
 // PostgresStatus contains status of the PostgreSQL cluster (running, creation failed etc.)
 type PostgresStatus string
+
+// LogicalBackup contains config of a k8s cron job responsible for running pg_dumpall
+type LogicalBackup struct {
+	EnableLogicalBackup   bool   `json:"enable_logical_backup,omitempty"`
```

Comment:
```
logical_backup:
    enabled: true
    schedule: "0 2 * * *"
```

Reply:
I am not sure if this should be nested in the manifest? what is more convenient? Similar to load balancers?

Reply:
at present we have both, for example volume size is nested:
```
volume:
    size: 1Gi
```
if we plan to add more options, we are imo better leaving it  nested